.github/workflows/build_and_test.yml

```diff
@@ -1112,14 +1112,12 @@ jobs:
         with:
           distribution: zulu
           java-version: ${{ inputs.java }}
-      - name: start minikube
-        run: |
-          # See more in "Installation" https://minikube.sigs.k8s.io/docs/start/
-          curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
-          sudo install minikube-linux-amd64 /usr/local/bin/minikube
-          rm minikube-linux-amd64
+      - name: Start Minikube
+        uses: medyagh/setup-minikube@v0.0.18
```

Comment:
why have you pinned the version to 0.0.18 and why not @latest 

with this `curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64` we had the latest always but now it its pinned.

Reply:
To @bjornjorgensen , we use `pinned` GitHub Action always.
```
$ git grep setup .github/workflows/build_and_test.yml
.github/workflows/build_and_test.yml:      uses: actions/setup-java@v4
.github/workflows/build_and_test.yml:      uses: actions/setup-python@v5
.github/workflows/build_and_test.yml:        uses: docker/setup-qemu-action@v3
.github/workflows/build_and_test.yml:        uses: docker/setup-buildx-action@v3
.github/workflows/build_and_test.yml:      uses: actions/setup-java@v4
.github/workflows/build_and_test.yml:      uses: actions/setup-java@v4
.github/workflows/build_and_test.yml:        # The followings are also used by `r-lib/actions/setup-r` to avoid
.github/workflows/build_and_test.yml:      uses: bufbuild/buf-setup-action@v1
.github/workflows/build_and_test.yml:      uses: actions/setup-python@v5
.github/workflows/build_and_test.yml:      uses: actions/setup-java@v4
.github/workflows/build_and_test.yml:      uses: actions/setup-java@v4
.github/workflows/build_and_test.yml:      uses: actions/setup-java@v4
.github/workflows/build_and_test.yml:      uses: actions/setup-java@v4
.github/workflows/build_and_test.yml:        uses: actions/setup-java@v4
.github/workflows/build_and_test.yml:        uses: actions/setup-node@v4
```

In addition, the GitHub Action version is irrelevant to the Minikube version. According to the log, it downloads the latest Minikube 1.33.1 like the picture.
![Screenshot 2024-08-21 at 13 25 32](https://github.com/user-attachments/assets/d064d2e6-1a47-4eae-90e7-b0998904fb48)

Reply:
It's the same like `setup-java`.

Reply:
Currently from market place: https://github.com/marketplace/actions/setup-minikube 0.0.18 is the latest.
I think pinning a version will be better to stabilized the test.